topic5/contracts/Service.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity ^0.4.23;
+
+contract Service {
+    
+    address public owner;
+    uint256 serviceCost = 1 ether;
```

Comment:
Could make `serviceCost` and `withdrawLimit` constants in order to save storage space.

Could make state variables public.